app/javascript/components/app-invoice.vue

```diff
@@ -2,16 +2,27 @@
   <div class="invoice">
     <div class="invoice__resume">
       <h3 class="invoice__title">Resumen</h3>
-      {{ invoice.amount || 0 }} Satoshis<br/>
-      ${{ invoice.clp || 0 }} CLP
+      <div
+        class="invoice__price"
+        v-bind:class="{ 'invoice__price--loading': loading}">
+          {{ invoice.amount || 0 }} Satoshis
+      </div>
+      <div
+        class="invoice__price"
+        v-bind:class="{ 'invoice__price--loading': loading}">
+          ${{ invoice.clp || 0 }} CLP
+      </div>
       <div class="invoice__copy" @click="copyPaymentRequest" v-if="invoice.payment_request">
         <div class="invoice__copy-value">{{ invoice.payment_request }}</div>
         <span class="invoice__copy-icon">
           <font-awesome-icon icon="clipboard" />
         </span>
       </div>
     </div>
-    <div class="invoice__info" v-if="invoice.payment_request">
+    <div class="invoice__loading" v-if="loading == 1">
```

Comment:
Attribute "if" should be on a new line                        vue/max-attributes-per-line